datadict/datadict-oci8.inc.php

```diff
@@ -185,32 +192,52 @@ function _CreateSuffix($fname,&$ftype,$fnotnull,$fdefault,$fautoinc,$fconstraint
 		return $suffix;
 	}
 
-/*
-CREATE or replace TRIGGER jaddress_insert
-before insert on jaddress
-for each row
-begin
-select seqaddress.nextval into :new.A_ID from dual;
-end;
-*/
+	/**
+	* Creates an insert trigger to emulate an auto-increment column
+	* in a table
+	*
+	* @param	str		$tabname		The name of the table
+	* @param	str[]	$tableoptions   Optional configuration items
+	*
+	* @return	str[]	The SQL statements to create the trigger
+	*/
```

Comment:
- add a space before the * on each line after the opening `/**` to align properly.
- use `string` not str for type
- use spaces for vertical alignment within comments